pkg/cloudprovider/aws/fleet/instance.go

```diff
@@ -66,6 +66,9 @@ func (p *InstanceProvider) Create(ctx context.Context,
 			},
 			Overrides: overrides,
 		}},
+		OnDemandOptions: &ec2.OnDemandOptionsRequest{
+			AllocationStrategy: aws.String(ec2.FleetOnDemandAllocationStrategyLowestPrice),
```

Comment:
Can we send both OnDemandOptions and SpotOptions? Does the API Complain? Since we can tolerate disruption, I think we want lowest price in both cases (for now).

Reply:
I think this will need more discussion, can we punt on this?

Reply:
Sure